app/app.go

```diff
@@ -352,6 +366,14 @@ func New(
 	return app
 }
 
+func GetDefaultBypassFeeMessages() []string {
+	return []string{
+		sdk.MsgTypeURL(&ibcchanneltypes.MsgRecvPacket{}),
```

Comment:
this will work considering the gas limitation?
https://github.com/cosmos/gaia/blob/v8.0.0-rc/x/globalfee/ante/fee.go#L67

Tx gas: (>400k)
https://www.mintscan.io/juno/txs/1DD7ADE4C7993554BB13BA83BBCE816287B786BDE784AC42B73F6EC2CF2C5181

Reply:
Probably not, good shout. I'll push a PR upstream to change via ante directly so we don't have to PR. Thanks! Missed this